test/MultisigControl_Unit_Tests.js

```diff
@@ -1254,7 +1255,7 @@ contract("MultisigControl -- Function: burn_nonce", (accounts) => {
 
         })
 
-        it("burnt nonce should become used (0030-ETHM-037, 0030-ETHM-057)", async () => {
+        it("burnt nonce should become used (0030-ETHM-037, 0030-ETHM-057, 0030-ETHM-057)", async () => {
```

Comment:
@C0deMunk33 
Should this be 0030-ETHM-056

Reply:
yes, I'll make that change